tests/unit/test_implementation.py

```diff
@@ -1,4 +1,36 @@
-# TODO
-##
-## _get_container_full_stem()
-## run()
+from pathlib import Path
+
+import pytest
+
+from linker.implementation import Implementation
+from linker.step import Step
+
+
+def test_implementation_instantiation(test_dir):
+    implementation_dir = (
```

Comment:
Rajan: after the refactor in the previous pr, make sure to add the directory-getter method test

Reply:
added as skipped/TODO test in replacement PR